docs/api/quickstart.md

```diff
@@ -0,0 +1,139 @@
+---
+title: API Quick Start Guide
+---
+
+The Rancher API is a programatic interface for your upstream and downstream clusters. This guide will help get you started on using the API from your client of choice.
```

Comment:
I think this statement is a bit inaccurate/misleading. We want to emphasize that this isn't a special/custom api, it's the kubernetes api which allows access to Rancher resources, which are kubernetes CRDs. This is important because there are a wide variety of tools (such as `client-go`, `kubectl`) which are compatible with the kubernetes api, and those tools should work with this approach.

```suggestion
Rancher's resources can be accessed through the Kubernetes API. This guide will help you get started on using this API (from your client of choice) as a Rancher user.
```